package-lock.json

```diff
@@ -6396,9 +6396,8 @@
       "integrity": "sha1-KzJxhOiZIQEXeyhWP7XnECrNDKk="
     },
     "node-gyp-build": {
-      "version": "3.5.0",
-      "resolved": "https://registry.npmjs.org/node-gyp-build/-/node-gyp-build-3.5.0.tgz",
-      "integrity": "sha512-qjEE8eIWVyqZhkAFUzytGpOGvLHeX5kXBB6MYyTOCPZBrBlsLyXAAzTsp/hWMbVlg8kVpzDJCZZowIrnKpwmqQ=="
+      "version": "github:puresec/node-gyp-build#e75daf49c212b2190d2ccadb3c01c0ad0459d54b",
```

Comment:
Diff of the actual fix: https://github.com/puresec/node-gyp-build/compare/master...puresec:dynamic_dep_fix#diff-168726dbe96b3ce427e7fedce31bb0bcR5

Reply:
Is this something which can be fixed in the upstream repo so there is no need to maintain a fork?

Reply:
I have created PR in the upstream repo: https://github.com/mafintosh/node-gyp-build/pull/10
We can wait for @mafintosh to merge it and publish a new package version.

Reply:
We are now using an upstream version of node-gyp-build

Reply:
@0xh0b0 thanks! That's awesome